tests/basics/builtin_next_arg2.py

```diff
@@ -0,0 +1,31 @@
+try:
+    print(next(iter([]), 42))
```

Comment:
Why print here?

Reply:
So it also serves as a test case. But it's maybe better to move that with the rest indeed, also it's not *that* different from the iter(range(0)) test so perhaps not needed as actual test at all?

Reply:
So, my point is that - if you test for exception, then print() call (and only print() call) is not really needed, and produces code/output noise. It's pretty minor issue of course.

Reply:
Yeah but details matter and without the print is better indeed.